test/core/test_util/fuzz_config_vars_helpers.h

```diff
@@ -15,14 +15,55 @@
 #ifndef GRPC_TEST_CORE_TEST_UTIL_FUZZ_CONFIG_VARS_HELPERS_H
 #define GRPC_TEST_CORE_TEST_UTIL_FUZZ_CONFIG_VARS_HELPERS_H
 
+// We can't currently compile fuzztest for Windows, so we separate
+// this code from the code to read the protobufs - allowing the latter
+// to also be used in regression tests that do run on Windows, and
+// reserving this code only for the actual fuzzers.
+
 #include <grpc/support/port_platform.h>
 #include <stdint.h>
 
 #include <string>
+#include <vector>
+
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_join.h"
+#include "fuzztest/fuzztest.h"
+#include "src/core/lib/debug/trace.h"
+#include "src/core/lib/experiments/experiments.h"
+#include "test/core/test_util/fuzz_config_vars.pb.h"
 
 namespace grpc_core {
 
-std::string ValidateExperimentsStringForFuzzing(uint64_t experiments);
+std::vector<std::string> ExperimentConfigChoices();
```

Comment:
Why not roll these helpers into fuzz_config_vars.{h,cc}? Do you expect we'd often need to use one without the other? They are almost always included together right now.

Reply:
Almost but not always.

fuzztest has some compilation issues on Windows, and with out strict build environment -- so anything that *can* operate as a unit test *cannot* include fuzztest.h right now, and so cannot include these helpers.